framework/test/unit/modules/pos/commands/report_misbehavior.spec.ts

```diff
@@ -528,6 +528,22 @@ describe('ReportMisbehaviorCommand', () => {
 
 			await expect(pomCommand.verify(context)).not.toReject();
 		});
+
+		it('should return error when generator of header1 is not a validator', async () => {
+			transactionParamsDecoded = {
+				header1: codec.encode(blockHeaderSchema, { ...header1, generatorAddress: senderAddress }),
+				header2: codec.encode(blockHeaderSchema, { ...header2, generatorAddress: senderAddress }),
```

Comment:
should the generatorAddress of header2 be updated to senderAddress? 🤔

Reply:
I think the description and expected error is not matching

Reply:
Yeah, the scenario as defined is `generator of header1 is not a delegate`

Reply:
I meant expecting to throw `Invalid block signature` is not checking the case `senderAddress` is not a validator?